doc/source/whatsnew/v0.16.0.txt

```diff
@@ -546,6 +546,37 @@ Bug Fixes
 - Bug in ``rank`` where comparing floats with tolerance will cause inconsistent behaviour (:issue:`8365`).
 - Fixed character encoding bug in ``read_stata`` and ``StataReader`` when loading data from a URL (:issue:`9231`).
 - Bug in adding ``offsets.Nano`` to other offets raises ``TypeError`` (:issue:`9284`)
+<<<<<<< HEAD
```

Comment:
Thanks for rebasing. You can remove this line, the one below it, and the one on 579. git puts these in when the same line has been edited on your branch and what you're rebasing onto.